Misc/NEWS.d/next/Core and Builtins/2021-01-02-23-32-55.bpo-41798.yZZWzQ.rst

```diff
@@ -0,0 +1,3 @@
+Allocate the ``expat_CAPI`` on the heap memory to avoid more module instances
+hold a single ``expat_CAPI`` object after pyexpat ported to multiphase
+initialization (:pep:`489`).
```

Comment:
I don't think that this change should be mentioned to end users. Only the actual conversion to multiphase init is worth to be mentionned (IMO).

Reply:
Removed.